TestProject/cellularAutomat/Board.php

```diff
@@ -175,4 +184,79 @@ function checkBoardOnSimilarities(): bool
             return false;
     }
 
+    /**
+     * @param $_height
+     */
+    function setHeight($_height)
+    {
+        $this->height=$_height;
+    }
+
+    /**
+     * @return int
+     */
+    function getHeight(): int
+    {
+        return $this->height;
+    }
+
+    /**
+     * @param $_width
+     */
+    function setWidth($_width)
+    {
+        $this->width=$_width;
+    }
+
+    /**
+     * @return int
+     */
+    function getWidth(): int
+    {
+        return $this->width;
+    }
+
+    /**
+     * @param $_board
+     */
+    function setBoard($_board)
+    {
+        $this->board[]=$_board;
+    }
+
+    /**
+     * @return array[]
+     */
+    function getBoard(): array
+    {
+        return $this->board;
+    }
+
+    /**
+     * @param $_x
+     * @param $_y
+     * @param $_value
+     * @return mixed
+     */
+    function setCell($_x, $_y, $_value)
+    {
+        return $this->board[$_x][$_y]= $_value;
+    }
+
+    /**
+     * @param $_lastGeneration
+     */
+    function setLastGeneration($_lastGeneration)
+    {
+        $this->lastGeneration[]=$_lastGeneration;
+    }
+
+    /**
+     * @return array
+     */
+    function getLastGeneration(): array
+    {
+        return $this->lastGeneration;
+    }
```

Comment:
Is this really necessary? What is the usecase of setting that from the outside? These are internal member variables that no one outside should know of.
Not every private member variable must be accessible from the outside. You have to think about what is necessary. If it becomes necessary at a later stage it is always possible to add getters/setters later on.